getting_started_vagrantbox.md

```diff
@@ -0,0 +1,85 @@
+## The goal of this guide 
+
+> :warning: Read the section `Description - what & why` in [README.md](/README.md) to get a quick introduction to what this repo is.
+
+The goal of this guide is to give an introduction to the [vagrant hashistack box](https://vagrantcloud.com/fredrikhgrelland/hashistack), its features, and how to use it.
+
+## What is the vagrant hashistack box
+The vagrant-hashistack box is a virtual machine built with [vagrant](https://www.vagrantup.com/), using the code from [this repository](https://github.com/fredrikhgrelland/vagrant-hashistack/). It aims to simulate a hashistack ecosystem, and with the help of built in switches and services it is able to be tweaked to a user's needs. The Norwegian Tax Administration is using this box as a part of their CI/CD pipeline to test terraform modules in their microservice architecture. 
```

Comment:
I would suggest do not mention this part
```suggestion
The vagrant-hashistack box is a virtual machine built with [vagrant](https://www.vagrantup.com/), using the code from [vagrant-hashistack](https://github.com/fredrikhgrelland/vagrant-hashistack/). It aims to simulate a hashistack ecosystem, and with the help of built in switches and services it is able to be tweaked to a user's needs. 
```

Reply:
Is it that bad to mention? :( It's so cool!

But I guess I can see what you mean

Reply:
Lets avoid risk